cluster/pre-existing/util.sh

```diff
@@ -44,19 +44,27 @@ function create-certs {
   rm /tmp/kubeconfig
 
   execute-cmd-on-pre-existing-master-with-retries "sudo cat /etc/kubernetes/admin.conf" > /tmp/kubeconfig
-  CA_CERT_BASE64=$(cat /tmp/kubeconfig | grep certificate-authority | awk '{print $2}' | head -n 1)
-  KUBELET_CERT_BASE64=$(cat /tmp/kubeconfig | grep client-certificate-data | awk '{print $2}' | head -n 1)
-  KUBELET_KEY_BASE64=$(cat /tmp/kubeconfig | grep client-key-data | awk '{print $2}' | head -n 1)
+  CA_CERT_BASE64=$(grep certificate-authority /tmp/kubeconfig | awk '{print $2}' | head -n 1)
+  export CA_CERT_BASE64
+  KUBELET_CERT_BASE64=$(grep client-certificate-data /tmp/kubeconfig | awk '{print $2}' | head -n 1)
+  KUBELET_KEY_BASE64=$(grep client-key-data /tmp/kubeconfig | awk '{print $2}' | head -n 1)
 
   # Local kubeconfig.kubemark vars
   KUBECFG_CERT_BASE64="${KUBELET_CERT_BASE64}"
+  export KUBECFG_CERT_BASE64
   KUBECFG_KEY_BASE64="${KUBELET_KEY_BASE64}"
+  export KUBECFG_KEY_BASE64
 
   # The pre-existing Kubernetes master already has these setup
   # Set these vars but don't use them
   CA_KEY_BASE64=$(dd if=/dev/urandom bs=128 count=1 2>/dev/null | base64 | tr -d "=+/" | dd bs=32 count=1 2>/dev/null)
+  export CA_KEY_BASE64
   MASTER_CERT_BASE64=$(dd if=/dev/urandom bs=128 count=1 2>/dev/null | base64 | tr -d "=+/" | dd bs=32 count=1 2>/dev/null)
+  export MASTER_CERT_BASE64
   MASTER_KEY_BASE64=$(dd if=/dev/urandom bs=128 count=1 2>/dev/null | base64 | tr -d "=+/" | dd bs=32 count=1 2>/dev/null)
+  export MASTER_KEY_BASE64
   KUBEAPISERVER_CERT_BASE64=$(dd if=/dev/urandom bs=128 count=1 2>/dev/null | base64 | tr -d "=+/" | dd bs=32 count=1 2>/dev/null)
+  export KUBEAPISERVER_CERT_BASE64
   KUBEAPISERVER_KEY_BASE64=$(dd if=/dev/urandom bs=128 count=1 2>/dev/null | base64 | tr -d "=+/" | dd bs=32 count=1 2>/dev/null)
+  export KUBEAPISERVER_KEY_BASE64
```

Comment:
I added `export` in many lines, because shellcheck said SC2034 as the below.
`SC2034: KUBEAPISERVER_KEY_BASE64 appears unused. Verify use (or export if used externally).`

Reply:
we should generally try to verify use so we can clean up things that are stale
https://cs.k8s.io/?q=KUBEAPISERVER_KEY_BASE64&i=nope&files=&repos=

Reply:
https://cs.k8s.io/?q=KUBEAPISERVER_KEY_BASE64&i=nope&files=&repos=kubernetes/kubernetes

Reply:
I don't think anything uses this.

Reply:
Thanks the advice.
I've checked all parameters on cs.k8s.io. KUBEAPISERVER_CERT_BASE64 may be used by only upgrade.sh.
https://cs.k8s.io/?q=KUBEAPISERVER_CERT_BASE64&i=nope&files=&repos=kubernetes-sigs/apiserver-builder-alpha

Should we leave this?
upgrade.sh is for GCE. Is it unnecessary because it is defined in other file in case of GCE?

Reply:
I suspect that's only an old copy in vendor in another repo, I don't see any in this repo so I think we can remove it.

you can see the TODO to remove these when 1.6 is not supported, which was long ago:
https://github.com/kubernetes-sigs/apiserver-builder-alpha/blob/master/cmd/vendor/github.com/kubernetes-incubator/reference-docs/vendor/k8s.io/kubernetes/cluster/gce/upgrade.sh#L125

Reply:
I see. As you told, https://github.com/kubernetes/kubernetes/pull/55324 removed the part of upgrade.sh.
I will remove KUBEAPISERVER_KEY_BASE64 and KUBEAPISERVER_CERT_BASE64.

Reply:
I've done that.